examples/k8s/k7e/community_day_2019_may/client-deployment.yaml

```
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: client
+  labels:
+    app: client
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: client
+  template:
+    metadata:
+      labels:
+        app: client
+    spec:
+      containers:
+        - name: client
+          image: gcr.io/spiffe-io/spire-agent:0.7.3
```

The server deployed with this example is v0.8.0 so I assume we want the agent to be too

```suggestion
          image: gcr.io/spiffe-io/spire-agent:0.8.0
```